flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/AsyncLookupJoinITCase.scala

```diff
@@ -268,6 +268,35 @@ class AsyncLookupJoinITCase(
     assertEquals(expected.sorted, sink.getRetractResults.sorted)
   }
 
+  @Test
+  def testAggAndAsyncLeftJoinWithTryResolveMode(): Unit = {
```

Comment:
Do we have any IT case to verify the change? this pr aims to support sync LookupJoin

Reply:
this case can cover the change, the legacy source can provide both sync and async functions, so it can fallback to sync lookup function with state.